infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Orchestrator.java

```diff
@@ -143,12 +147,29 @@ synchronized void dispatch(CorfuPayloadMsg<OrchestratorMsg> msg,
     void run(@Nonnull IWorkflow workflow) {
         CorfuRuntime rt = null;
         try {
-            Layout currLayout = getRuntime.call().layout.get();
-            String servers = String.join(",", currLayout.getLayoutServers());
-            rt = new CorfuRuntime(servers)
-                    .setCacheDisabled(true)
-                    .setLoadSmrMapsAtConnect(false)
-                    .connect();
+            getRuntime.call().invalidateLayout();
+            Layout currLayout = getRuntime.call().getLayoutView().getLayout();
+
+            List<NodeLocator> servers = currLayout.getAllServers().stream()
+                    .map(NodeLocator::parseString)
+                    .collect(Collectors.toList());
+
+            CorfuRuntimeParameters currParams = getRuntime.call().getParameters();
+            CorfuRuntimeParameters params = CorfuRuntimeParameters.builder()
+                    .cacheDisabled(true)
```

Comment:
Sorry, missed this on my first pass. Can you add comments as to why you're disabling the cache, etc. Why do the parameters differ?

Reply:
Added comments.

The workflows don't use the cache and just to make sure that the orchestrator doesn't use extra memory. 
It also doesn't open streams, so it doesn't need the fast loader.